packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1800,6 +1805,22 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
     _markDirty();
   }
 
+  /// Whether this user can interact with this node in assistive technologies.
+  ///
+  /// This node can still be receive accessibility focus even if this is true.
+  /// Setting this true prevents user from activating pointer related
+  /// [SemanticsAction]s, such as [SemanticsAction.tap] or
+  /// [SemanticsAction.longPress].
+  bool get isUserInteractionsBlocked => _isUserInteractionsBlocked;
```

Comment:
was curious if there is a better name

Reply:
Maybe: `blocksUserActions` or `blocksSemanticsActions` (in general, maybe we should stick with the "actions" terminology throughout this PR to make it clear that this relates to SemanticsActions)?

Reply:
I agree to change this to "Actions". but I may keep it past tense since it is already a done deal in SemanticsNode level. Present tense kinda implies it will do something that it hasn't done.